server/testutils/factories/newPlacementRequestBookingConfirmation.ts

```diff
@@ -0,0 +1,17 @@
+import { Factory } from 'fishery'
+import { faker } from '@faker-js/faker/locale/en_GB'
+
+import type { NewPlacementRequestBookingConfirmation } from '@approved-premises/api'
+
+import { DateFormats } from '../../utils/dateUtils'
+
```

Comment:
would it be worth having this in the same file as newPlacementRequestBookingFactory?